protocol/contracts/beanstalk/barn/UnripeFacet.sol

```diff
@@ -236,10 +227,10 @@ contract UnripeFacet is ReentrancyGuard {
     /**
      * @notice Returns the % penalty of Chopping an Unripe Token into its Ripe Token.
      * @param unripeToken The address of the Unripe Token.
-     * @return penalty The penalty % of Chopping.
+     * @return penalty The penalty % of Chopping derived from %Recapitalized^2.
      */
     function getPercentPenalty(address unripeToken) external view returns (uint256 penalty) {
-        return LibUnripe.getRecapPaidPercentAmount(getRecapFundedPercent(unripeToken));
+        return (s.u[unripeToken].balanceOfUnderlying ** 2).mul(LibUnripe.DECIMALS).div(IERC20(unripeToken).totalSupply() ** 2);
```

Comment:
This logic is duplicated - once in LibUnripe and once here. When writing code, if code is written twice, it increases the potential for future mistakes (another change in a penalty % would result in the developer needing to know that the logic is implmented twice). Thus, reduce duplicated logic whenever possible. 
```suggestion
        return libUnripe._getUnderlying(unripeToken, LibUnripe.DECIMALS, IERC20(unripeToken).totalSupply())
```

Reply:
Was actually thinking about this when i was writing it. Appreciate the feedback will try to remember it as much as possible moving forward. fixed here [6f02dd2](https://github.com/BeanstalkFarms/Beanstalk/pull/802/commits/6f02dd2019cec27a2f721b425e5ec1b1ad46edba)